packages/ui/VirtualList/VirtualListBase.js

```diff
@@ -213,6 +213,14 @@ class VirtualListBase extends Component {
 		 */
 		overhang: PropTypes.number,
 
+		/**
+		 * Support additional padding size
+		 *
+		 * @type {Boolean}
+		 * @private
+		 */
+		padded: PropTypes.bool,
```

Comment:
Don't we need some 'padding' prop rather than 'padded'? Theme libs require different padding values from each other.